mountpoint-s3-crt/src/http/request_response.rs

```diff
@@ -243,15 +243,15 @@ impl<'a> Iterator for HeadersIterator<'a> {
 
 /// A single HTTP message, initialized to be empty (i.e., no headers, no body).
 #[derive(Debug)]
-pub struct Message<'a> {
+pub struct Message {
     /// The pointer to the inner `aws_http_message`.
     pub(crate) inner: NonNull<aws_http_message>,
 
     /// Input stream for the body of the http message, if present.
-    body_input_stream: Option<InputStream<'a>>,
+    body_input_stream: Option<AsyncInputStream>,
```

Comment:
Can we just delete the old InputStream implementation now? We're not using it anywhere, right?